core/src/test/scala/org/apache/spark/deploy/history/HybridStoreSuite.scala

```diff
@@ -103,6 +104,13 @@ abstract class HybridStoreSuite extends SparkFunSuite with BeforeAndAfter with T
   }
 
   test("test basic iteration") {
+
+    def firstInView[T](view: KVStoreView[T]): T = {
```

Comment:
Do you want to call this "head"?

Reply:
[70528b6](https://github.com/apache/spark/pull/36237/commits/70528b6d0af97636845196e4a764fdce6d41e2db) fix this